src/components/casestudyandcoretools/Casestudy.tsx

```diff
@@ -0,0 +1,40 @@
+import React from "react";
```

Comment:
change this filename to match the name of the component such as Casestudy to CaseStudy

Reply:
Done